le-parole-della-pubblica-amministrazione/e.rst

```diff
@@ -1,13 +1,13 @@
 E
 =
 
-eccetera, ecc. o etc.
-     Tutte e tre le forme sono corrette. Se si tratta di elenchi, scegli la forma abbreviata “ecc.”. Qualunque versione tu abbia scelto, ricorda di usarla uniformemente in tutto il testo.
+eccetera, ecc., et cetera, etc.
+     Tutte le forme sono corrette. La forma eccetera (e l’abbreviazione ecc.) è preferibile e risulta più comune rispetto alla formula latina *et cetera* (e l’abbreviazione etc.)che è molto utilizzata da americani e anglosassoni. Se si tratta di elenchi, scegli la forma abbreviata "ecc." che può essere preceduta (o meno) da virgola. Qualunque versione tu abbia scelto, ricorda di usarla uniformemente in tutto il testo.
```

Comment:
```suggestion
eccetera, ecc., et cetera, etc.
     Tutte le forme sono corrette, ma preferisci la forma “eccetera”. Se si tratta di elenchi, scegli la forma abbreviata “ecc.” che può essere preceduta (o meno) da virgola. Qualunque versione tu abbia scelto, ricorda di usarla uniformemente in tutto il testo.
```

Qui starei più sintetico (il fatto che sia usata da anglofoni è una considerazione che in fondo non ci interessa) e darei sempre del tu, come nel resto del writing-toolkit ed essere prescrittivo, lasciando le motivazioni a queste review o alle discussioni nelle issue.

@gianvitofanelli @danielaiozzo la virgola facoltativa ci torna?